src/Data/AuditContext.cs

```diff
@@ -0,0 +1,19 @@
+using IdentityServerConfig.Models;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace IdentityServerConfig.Data;
+
+public class AuditContext : DbContext
+{
+    public AuditContext(DbContextOptions<AuditContext> options) : base(options)
+    {
+    }
+    
+    public DbSet<AuditLog> AuditLogs { get; set; }
+    
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+       modelBuilder.Entity<AuditLog>().ToTable("AuditLog", "IdentityServerConfig");
```

Comment:
Can we make the schema name configurable somehow?

Reply:
Why does it need to be configurable? By keeping it set as IdentityServerConfig it would be more recognizable as belonging to IdentityServerconfig. This is lost when the schema name can be changed.